consoleApp/src/main/java/com/infoshare/Menu.java

```diff
@@ -38,7 +40,8 @@ private void handleUserChoice () {
                         break;
                     }
                     case 2: {
-                        System.out.println("Zgłoś osobę potrzebującą pomocy");
+                        NeedRequestService needRequestService=new NeedRequestService();
+                        needRequestService.createNeedRequest();
```

Comment:
Jeżeli nie będziemy później używać needRequestService to można w jednej linijce zamknąć